docs/reference/modules/snapshots.asciidoc

```diff
@@ -62,23 +62,26 @@ can be obtained using the following command:
 
 [source,js]
 -----------------------------------
-GET /_snapshot/repo*,*backup*
+GET _snapshot/repo*,*backup*
 -----------------------------------
+// CONSOLE
 
 If a repository name is not specified, or `_all` is used as repository name Elasticsearch will return information about
 all repositories currently registered in the cluster:
 
 [source,js]
 -----------------------------------
-GET /_snapshot
+GET _snapshot
 -----------------------------------
+// CONSOLE
 
 or
 
 [source,js]
 -----------------------------------
-GET /_snapshot/_all
+GET _snapshot/_all
 -----------------------------------
+// CONSOLE
```

Comment:
Do these pass? Like, I expect most of them to 404 unless you do something to make sure the repository exists.